source/scpp/src/scp/SCPDriver.h

```diff
@@ -100,6 +100,14 @@ class SCPDriver
     // should be flooded to the network.
     virtual void emitEnvelope(SCPEnvelope const& envelope) = 0;
 
+    // `getNodeID` gets the UTXO hash for a node with the `node_index`
+    // at the height that is handed in as `slotIndex`.
+    virtual Hash getNodeID(uint64 slotIndex, uint64 nodeIndex) = 0;
+
+    // `getNodeIndex` gets the index of the UTXO hash for a node with
+    // the `node_id` at the height that is handed in as `slotIndex`.
+    virtual uint64 getNodeIndex(uint64 slotIndex, Hash nodeID) = 0;
+
```

Comment:
I don't see this being used ?

Reply:
I made it for the case where a `NominationProtocol` could refer to a `NodeID` or an index. But will remove if unnecessary.